Tests/XCUITests/FakespotTests.swift

```diff
@@ -0,0 +1,38 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+import XCTest
+import Shared
```

Comment:
`import Shared` is giving me a new warning but the identifier does not work without this import. 🤔 
```
⚠️  /Users/vagrant/git/Tests/XCUITests/FakespotTests.swift:5:8: implicit import of bridging header 'Shared-Bridging-Header.h' via module 'Shared' is deprecated and will be removed in a later version of Swift
import Shared
```

Reply:
@clarmso we should try to fix this warning before merging, otherwise we will be close to threshold again...